airbyte-cdk/python/airbyte_cdk/sources/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .abstract_source import AbstractSource
 from .config import BaseConfig
 from .source import Source
+from .types import Config, Record
```

Comment:
Do we need to add these to `__all__`?

Reply:
I kinda think we can go either way with these. We could go all in on the newly defined types. And try to normalize or encourage our Python sources to use stronger types like the `StreamSlice` concept. Or like Ella mentioned we just don't expose them. Until a week ago these didn't even exist in Python CDK so most connectors probably don't refrerence them

Two questions:
- Do we want to expose the `StreamSlice` concept which might help add some more sanity to our Python susbstreams?
- Right now our method interfaces don't account for a lot of these types so it's not intuitive to use the new classes. But if we're making the 1.0.0 bump, would it be a good idea to improve the method interfaces?